Resources/Prototypes/Roles/Jobs/Fun/misc_startinggear.yml

```diff
@@ -47,6 +47,8 @@
     ears: ClothingHeadsetGrey
     pocket1: VehicleSkeletonMotorcycle
     pocket2: VehicleKeySkeletonMotorcycle
+  inhand:
+    left: JetpackBlackFilled
```

Comment:
you seem to have put this in the skeleton biker gear rather than the ninja gear.